Source/carthage/Update.swift

```diff
@@ -53,20 +58,23 @@ public struct UpdateOptions: OptionsType {
 		}
 	}
 
-	public static func create(configuration: String) -> BuildPlatform -> Bool -> Bool -> Bool -> CheckoutOptions -> UpdateOptions {
-		return { buildPlatform in { verbose in { checkoutAfterUpdate in { buildAfterUpdate in { checkoutOptions in
-			return self.init(checkoutAfterUpdate: checkoutAfterUpdate, buildAfterUpdate: buildAfterUpdate, configuration: configuration, buildPlatform: buildPlatform, verbose: verbose, checkoutOptions: checkoutOptions)
-		} } } } }
+	public static func create(configuration: String) -> BuildPlatform -> Bool -> Bool -> Bool -> CheckoutOptions -> String -> UpdateOptions {
+		return { buildPlatform in { verbose in { checkoutAfterUpdate in { buildAfterUpdate in { checkoutOptions in { dependenciesToUpdate in
+			let dependenciesToUpdate: [String]? = dependenciesToUpdate.isEmpty ? nil : dependenciesToUpdate.split()
+			return self.init(checkoutAfterUpdate: checkoutAfterUpdate, buildAfterUpdate: buildAfterUpdate, configuration: configuration, buildPlatform: buildPlatform, verbose: verbose, checkoutOptions: checkoutOptions, dependenciesToUpdate: dependenciesToUpdate)
+		} } } } } }
 	}
 
 	public static func evaluate(m: CommandMode) -> Result<UpdateOptions, CommandantError<CarthageError>> {
+		let dependenciesToUpdate: Result<String, CommandantError<CarthageError>> = m <| Option(defaultValue: "", usage: "the dependency names to update and build separated by commas or spaces")
```

Comment:
We really shouldn't require dependency names to be a single argument—they should be multiple arguments: `carthage update X Y`. This will probably require changes to Commandant.